specification/offazurespringboot/resource-manager/Microsoft.OffAzureSpringBoot/preview/2023-01-01-preview/springbootdiscovery.json

```diff
@@ -858,17 +893,72 @@
           }
         },
         "x-ms-examples": {
-          "springbootapps_Update_MaximumSet_Gen": {
-            "$ref": "./examples/springbootapps_Update_MaximumSet_Gen.json"
+          "springbootapps_Delete_MaximumSet_Gen": {
+            "$ref": "./examples/springbootapps_Delete_MaximumSet_Gen.json"
           },
-          "springbootapps_Update_MinimumSet_Gen": {
-            "$ref": "./examples/springbootapps_Update_MinimumSet_Gen.json"
+          "springbootapps_Delete_MinimumSet_Gen": {
+            "$ref": "./examples/springbootapps_Delete_MinimumSet_Gen.json"
           }
         },
         "x-ms-long-running-operation": true,
         "x-ms-long-running-operation-options": {
           "final-state-via": "location"
         }
+      },
+      "patch": {
+        "tags": [
+          "springbootapps"
+        ],
+        "operationId": "springbootapps_Update",
+        "description": "Update a springbootapps resource.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SiteNameParameter"
+          },
+          {
+            "$ref": "#/parameters/springbootappsNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "springbootapps",
+            "description": "Update a springbootapps payload.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/springbootappsPatch"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updates the springbootapps resource.",
+            "schema": {
+              "$ref": "#/definitions/springbootappsModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "springbootapps_Update_MaximumSet_Gen": {
+            "$ref": "./examples/springbootapps_Update_MaximumSet_Gen.json"
+          },
+          "springbootapps_Update_MinimumSet_Gen": {
+            "$ref": "./examples/springbootapps_Update_MinimumSet_Gen.json"
+          }
+        }
       }
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OffAzureSpringBoot/springbootsites/{siteName}/springbootapps": {
```

Comment:
>readOnly [](http://example.com/codeflow?start=11&length=8)

This is a breaking change, will add the label.

Reply:
The provisionState should be readonly by the validation rules

Reply:
It's still a breaking change, but should be safe for this property.